gender_novels/novel.py

```diff
@@ -120,12 +121,13 @@ def __str__(self):
         return name
 
 
-    def _load_novel_text(self):
-        """Loads the text of a novel and removes boilerplate at the beginning and end
+    def load_novel_text(self):
```

Comment:
The function should still be private (starting with an underscore). The way to access the text should be via the Novel.text attribute.